device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -26,6 +26,9 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```

Comment:
What's the difference between importing CustomLogger or slf4j classes in each file? slf4j library is highly mature and quite a lot of professionell java project are doing it that way. Using custom logging classes is not best practice.